src/Microsoft.Framework.Runtime.Hosting.Interfaces/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+    "version": "1.0.0-*",
+    "description": "Interfaces to take advantage of capabilities exposed by the runtime",
+    "dependencies": {
+    },
+
+    "frameworks" : {
+        "aspnet50" : { 
+            "dependencies": {
+            }
+        },
+        "aspnetcore50" : { 
+            "dependencies": {
+                "System.Runtime": "4.0.20-beta-22606"
```

Comment:
most of this code is copy-pasta, I'll go through and touch up the various notes you've made.